app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt

```diff
@@ -70,13 +70,12 @@ class TalkTopicActivity : BaseActivity(), LinkPreviewDialog.Callback {
         topicId = intent.extras?.getInt(EXTRA_TOPIC, -1)!!
 
         L10nUtil.setConditionalLayoutDirection(binding.talkRefreshView, pageTitle.wikiSite.languageCode())
+        binding.talkRefreshView.setColorSchemeResources(ResourceUtil.getThemedAttributeId(this, R.attr.colorAccent))
 
         binding.talkRecyclerView.layoutManager = LinearLayoutManager(this)
         binding.talkRecyclerView.addItemDecoration(DrawableItemDecoration(this, R.attr.list_separator_drawable, drawStart = false, drawEnd = false))
         binding.talkRecyclerView.adapter = TalkReplyItemAdapter()
 
-        L10nUtil.setConditionalLayoutDirection(binding.talkRefreshView, pageTitle.wikiSite.languageCode())
```

Comment:
Was removing this intentional?

Reply:
Yes, it is. You can see the same code above and this one is redundant.